README.md

```diff
@@ -1 +1,29 @@
-# nokia-basic-dci-lab
\ No newline at end of file
+# nokia-basic-dci-lab
+
+A basic Datacenter Interconnect (DCI) lab for with leaf/spine switches powered by SR Linux and DC Gateway and P routers powered by Nokia 7750 SR OS.
+
+## Deploy on c9s
+
+Setting up additional volume mount to get access to the license file
+
+```
+alias clabverter='sudo docker run --user $(id -u) \
+    -v /opt/nokia:/opt/nokia \
+    -v $(pwd):/clabernetes/work --rm \
+    ghcr.io/srl-labs/clabernetes/clabverter:latest'
+```
+
+and then run
+
+```
+clabverter --stdout --naming non-prefixed --disableExpose true \
```

Comment:
`--disableExpose true` should just be `--disableExpose` fyi!

Reply:
roger that